### DISCUSSION TITLE
什么时候更新迷识之径的遗器本呀

### DISCUSSION THREAD

**Discussion by lzstellata** [moesnow/March7thAssistant#366] [Q&A]
什么时候更新迷识之径的遗器本呀

**Comment by Shiratsuk1** [comment]
使用源码部署已经有新本的数据了

**Reply by lzstellata** [reply]
使用源码部署之后确实有新本的选项，但是点击确定之后又变回之前的了·-

**Reply by lzstellata** [reply]
啊不在客户端里设置就可以！我搞定了（鞠躬

**Comment by moesnow** [comment]
急用替换 `assets/config/instance_names.json` 文件就可以了，新副本更新不及时也可以自己手动添加内容
